cloudify/ctx_wrappers/ctx-py.py

```diff
@@ -34,7 +41,7 @@ def check_output(*popenargs, **kwargs):
 
 
 def unicode_to_string(text):
```

Comment:
the whole premise of this function is wrong, because we should be dealing with unicode internally, and not bytes, so there shouldn't be a need for a dump-to-bytes util. But for now, i'm not making changes bigger than necessary